gradle.properties

```diff
@@ -111,12 +111,23 @@ artifactRedirecting.androidx.collection.version=1.4.0
 artifactRedirecting.androidx.annotation.version=1.8.0-alpha02
 # TODO: per-target-versioning for annotation is temporary, delete it and keep the line above (after stable 1.8.0)
 artifactRedirecting.androidx.annotation.targetVersions=jvm=1.7.1,default=1.8.0-alpha02
-# FIXME: lifecycle-common, lifecycle-runtime should redirect to 2.7.0
-# FIXME: lifecycle-runtime-compose should redirect to <= 2.8.0-alpha02
+# Lifecycle group and modules versions are set according to https://youtrack.jetbrains.com/issue/COMPOSE-1306
 artifactRedirecting.androidx.lifecycle.version=2.8.0-alpha04
+artifactRedirecting.androidx.lifecycle.lifecycle-runtime-compose.version=2.8.0-alpha02
+# FIXME: in 1.6.10 we should publish our artifacts of  lifecycle-common and lifecycle-runtime with 2.7.0 version and redirect only for android:
+artifactRedirecting.androidx.lifecycle.lifecycle-common.version=2.8.0-alpha04
+artifactRedirecting.androidx.lifecycle.lifecycle-runtime.version=2.8.0-alpha04
 artifactRedirecting.androidx.navigation.version=2.8.0-alpha05
 artifactRedirecting.androidx.savedstate.version=1.2.1
 
+# For the purpose of substituteForRedirectedPublishedDependencies (look for it in the code)
+artifactRedirecting.modules-for-knative-manifest=\
```

Comment:
It's a small refactoring for k/native manifest dependencies redirection

Reply:
Why do we need to specify `version` here? Don't we already have a module -> version mapping?

Reply:
It's possible to avoid it. I wanted to make it more explicit, so we explicitly declare it.

Reply:
In the future it can be reduced to:

```
:annotation:annotation=artifactRedirecting.androidx.annotation.version,\
  :collection:collection=artifactRedirecting.androidx.collection.version,\
  :lifecycle:lifecycle-common=artifactRedirecting.androidx.lifecycle.version,\
  :lifecycle:lifecycle-runtime=artifactRedirecting.androidx.lifecycle.version,\
  :lifecycle:lifecycle-viewmodel=artifactRedirecting.androidx.lifecycle.version
```

Reply:
I see that as an extra point for misconfiguration but don't see the cases when it will be different to existing matching. out of scope for this PR anyway